cmd/export_liquidity_pools.go

```diff
@@ -24,67 +22,46 @@ should be used in an initial data dump. In order to get liqudity pools informati
 the export_ledger_entry_changes command.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		cmdLogger.SetLevel(logrus.InfoLevel)
-		endNum, useStdout, strictExport, isTest := utils.MustCommonFlags(cmd.Flags(), cmdLogger)
+		endNum, strictExport, isTest := utils.MustCommonFlags(cmd.Flags(), cmdLogger)
+		cmdLogger.StrictExport = strictExport
 		env := utils.GetEnvironmentDetails(isTest)
 		path := utils.MustBucketFlags(cmd.Flags(), cmdLogger)
+		gcsBucket, gcpCredentials := utils.MustGcsFlags(cmd.Flags(), cmdLogger)
 
 		pools, err := input.GetEntriesFromGenesis(endNum, xdr.LedgerEntryTypeLiquidityPool, env.ArchiveURLs)
 		if err != nil {
 			cmdLogger.Fatal("could not read accounts: ", err)
 		}
 
-		var outFile *os.File
-		if !useStdout {
-			outFile = mustOutFile(path)
-		}
-
-		failures := 0
-		numBytes := 0
+		outFile := mustOutFile(path)
+		numFailures := 0
+		totalNumBytes := 0
 		for _, pool := range pools {
 			transformed, err := transform.TransformPool(pool)
 			if err != nil {
 				if strictExport {
 					cmdLogger.Fatal("could not transform pool", err)
 				} else {
 					cmdLogger.Warning("could not transform pool", err)
-					failures++
+					numFailures++
 					continue
 				}
 			}
 
```

Comment:
Was this missed in refactor? Does not match the rest of the `cmd` utility files

Reply:
Thanks for catching this. I have changed it to look like the rest.